### PR TITLE
add blog post about 2018 spring bug bbq

### PR DESCRIPTION
**Pull request by fmichonneau**
please do not merge before it's been reviewed by @ErinBecker

### CODE REVIEW

File: _posts/2018-03-07-bug-bbq.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau", "Erin Becker"]
+---
+
+
+The Carpentries are preparing to publish the **Social Sciences** and the **Geospatial** curricula on [Zenodo](https://zenodo.org/) by mid-May. This will be the first
```
Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: indicate that these lessons are DC

Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: Social sciences lessons are scheduled for publication in April, Geospatial in June.

File: _posts/2018-03-07-bug-bbq.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* [Python](https://github.com/datacarpentry/python-socialsci)
+* [OpenRefine](https://github.com/datacarpentry/openrefine-socialsci)
+* [R](https://github.com/datacarpentry/r-socialsci)
+
+
+## Get involved!
+
+If you've made a contribution to one of these lessons, you're already an author. Help make sure the final product is polished and complete by getting involved in the lesson release.
+
+We are organizing a Bug BBQ to prepare these lessons for release. The main goal for the Bug BBQ is to get the Geospatial and Social Science lessons ready for release. However, if you are a Maintainer for another lesson, and you are available and interested in getting some extra eyes on your lessons, let [François](mailto:francois@carpentries.org) know and we'll include your repository on the Bug BBQ website.
```
Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: I like this language.

Comment (ErinBecker) [reply _posts/2018-03-07-bug-bbq.md]: Can we also follow up with specific Maintainers and ask if they want to be included. I'm thinking of:

- DC Genomics R (not published)
- DC Genomics Project organization and management
- SWC Plotting and Programming in Python (not published?)

File: _posts/2018-03-07-bug-bbq.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+## Bug BBQ details
+
+The Bug BBQ will start on April 12th, 2018 at [9am](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Spring+2018+Carpentries+Bug+BBQ+--+Start&iso=20180412T13) Eastern Time (1pm UTC) and end on April 13th, 2018 at [5pm](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Spring+2018+Carpentries+Bug+BBQ+--+End&iso=20180414T00) Pacific Time (midnight on April 14th UTC). (Click on the links to see these times for your time zones)
```
Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: Eastern Time USA, Pacific Time USA

File: _posts/2018-03-07-bug-bbq.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+During a Bug BBQ, the community gathers online to squash and grill as many bugs as possible to make our lessons polished and ready to be officially released.
+
+This is a distributed event, so you don't have to go anywhere. We'll use GitHub, Slack, and a website to get organized. If you are part of a local Carpentries community with several people interested in taking part in the Bug BBQ, feel free to organize a real BBQ to feed the crowd. If you plan on getting together, let us know by opening an issue! We'll add you to the website so other people can join you.
+
+## How long should I attend?
+
+The event lasts almost 36 hours to accommodate working hours across the globe. We are a global community and we want everyone to have a chance to participate. Feel free to participate for as little or as long as you want. However, note that contributions made when sleep deprived are rarely the best ones.
+
+If you are a maintainer, please coordinate with the other maintainers for your lesson to be ready to review, and provide feedback on the issues and pull requests that you will receive during the Bug BBQ.
```
Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: Please capitalize Maintainer (and other community roles)

File: _posts/2018-03-07-bug-bbq.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Who is the Bug BBQ for?
+
```
Comment (ErinBecker) [inline _posts/2018-03-07-bug-bbq.md]: If you have questions contact . . . ?

**Comment by ErinBecker** [comment]
@fmichonneau - I've added some comments and suggestions. Please feel free to merge after making those changes.

File: _posts/2018-03-07-bug-bbq.md (hunk 2 above)
Comment (fmichonneau) [reply _posts/2018-03-07-bug-bbq.md]: yes, I'll follow up with them individually tomorrow.

**Comment by fmichonneau** [comment]
I updated the text, please merge if it looks good to you.

**Comment by ErinBecker** [comment]
Looks great, thanks @fmichonneau! Could you please also post on SWC and send out a tweet?

**Pull request merged**